components/org.wso2.carbon.identity.recovery/src/test/java/org/wso2/carbon/identity/recovery/password/NotificationPasswordRecoveryManagerTest.java

```diff
@@ -80,6 +90,10 @@ public class NotificationPasswordRecoveryManagerTest {
     private static final String TRUE_STRING = "true";
     private static final int TENANT_ID = 1234;
 
+    public static final String PRE_UPDATE_PASSWORD_ACTION_EXECUTION_FAILED = "USER-ACTION-PRE-UPDATE-PASSWORD-60001";
+    public static final String EXTENSION_ERROR_MESSAGE = "invalid_password";
+    public static final String EXTENSION_ERROR_DESCRIPTION = "Invalid password format";
```

Comment:
Does these need to be public ?

Reply:
Fixed with d8a7e0f199ca0dd41ddeb24807f49e89527d6b61